posthog/models/web_preaggregated/sql.py

```diff
@@ -13,10 +11,10 @@ def is_eu_cluster() -> bool:
     return getattr(settings, "CLOUD_DEPLOYMENT", None) == "EU"
 
 
-def TABLE_TEMPLATE(table_name, columns, order_by, on_cluster=True, force_unique_zk_path=False, replace=False):
+def TABLE_TEMPLATE(table_name, columns, order_by, on_cluster=True, replace=False, zk_path=None):
```

Comment:
It could be simpler to just add one parameter more, but let's just not allow it to avoid a "temporary solution that stays around forever".